src/node_dotenv.cc

```diff
@@ -102,8 +102,11 @@ MaybeLocal<Object> Dotenv::ToObject(Environment* env) const {
   return scope.Escape(result);
 }
 
-// Removes space characters (spaces, tabs and newlines) from
-// the start and end of a given input string
+// Removes leading and trailing spaces from a string_view.
+// Returns an empty string_view if the input is empty.
+// Example:
+//   trim_spaces("  hello  ") -> "hello"
+//   trim_spaces("") -> ""
```

Comment:
Not new to this PR but it occurs to me that this function overlooks the often-overlooked vertical tab (0x0B).

Reply:
I maybe can help on that, but I don't know what your are talking about